homework_8.py

```diff
@@ -0,0 +1,65 @@
+"""My Homework 8"""
+
+# "Task #1"
+
+
+def is_increasing_sequence(sequence):
+    """Check an increasing sequence"""
+    sequence = tuple(sequence)
```

Comment:
`sequence` variable comes as a list, you do not need to convert to tuple or any other type. Work with list